packages/mdc-ripple/_mixins.scss

```diff
@@ -195,12 +212,20 @@
 
   &.mdc-ripple-upgraded--unbounded#{$pseudo} {
     top: 0;
+
+    /* @alternate */
```

Comment:
For readability, I would suggest using the same style you have above:

```css
/* @alternate */
top: 0;
top: var(--mdc-ripple-top, 0);

/* @alternate */
left: 0;
left: var(--mdc-ripple-left, 0);

...
```

Or you could do something like this:

```css
/* @alternate */ top: 0;
/* @alternate */ top: var(--mdc-ripple-top, 0);

/* @alternate */ left: 0;
/* @alternate */ left: var(--mdc-ripple-left, 0);

...
```

Personally, I tend to prefer the first one. Since we only have two alternate values, we can get away with only having a single annotation.

Ditto for the other `@alternate` annotations in this file.